pghoard/pghoard.py

```diff
@@ -455,11 +455,18 @@ def main(args=None):
         description="postgresql automatic backup daemon")
     parser.add_argument("--version", action="version", help="show program version",
                         version=version.__version__)
-    parser.add_argument("config", help="configuration file")
+    parser.add_argument("--config", help="configuration file path", default=os.environ.get("PGHOARD_CONFIG"))
```

Comment:
What's the point of PGHOARD_CONFIG env?  In which cases does it make operation easier?  I can see why you'd prefer --config flag over positional arguments though